### ISSUE TITLE
Crashed trying to add ipv6 option for existing droplet

### ISSUE THREAD

**Issue by holms** [terraform-providers/terraform-provider-digitalocean#116]
### Terraform Version
```
holms@debian ~/D/c/l/a/p/terraform> terraform -v
Terraform v0.11.7
+ provider.ansible v0.0.4
+ provider.aws v1.27.0
+ provider.digitalocean v0.1.3
+ provider.mysql v1.1.0

Your version of Terraform is out of date! The latest version
is 0.11.8. You can update by downloading from www.terraform.io/downloads.html

```
### Affected Resource(s)
- digitalocean_droplet

### Terraform Configuration Files
```hcl
resource "digitalocean_droplet" "etcd-master" {

    name                = "etcd-master-${count.index}"
    image               = "${var.image}"
    region              = "${var.region}"
    size                = "${var.etcd_memory_master}"
    ssh_keys            = "${var.ssh_keys}"
    count               = "${var.etcd_count_master}"
    private_networking  = 1
    ipv6                = 1

    connection {
      user = "root"
      agent = 1
    }

    provisioner "remote-exec" {
      inline = [
        "apt-get update",
        "apt-get install -y python",
      ]
    }

}

resource "digitalocean_droplet" "etcd-slave" {
    name                = "etcd-slave-${count.index}"
    image               = "${var.etcd_image}"
    region              = "${var.region}"
    size                = "${var.etcd_memory_slave}"
    ssh_keys            = "${var.ssh_keys}"
    count               = "${var.etcd_count_slave}"
    private_networking  = 1
    ipv6                = 1

    connection {
      user = "root"
      agent = 1
    }

    provisioner "remote-exec" {
      inline = [
        "apt-get update",
        "apt-get install -y python",
      ]
    }
}

```

### Debug Output
https://gist.github.com/holms/704cb744404a7a7a0a44362b1e2c4dbd

### Panic Output
https://gist.github.com/holms/704cb744404a7a7a0a44362b1e2c4dbd

### Expected Behavior
It should've been add ipv6 option for droplet without any problems

### Actual Behavior
Terraform crashed. When launched again, droplets has been created from scratch. Terraform just "forgot" that it had it state, which is sad, if this would happen on real production I'd have really bad time to fixing this

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
As you can see I've added AAAA records, but I had to comment them out, as they were executing first somehow. 

This is actually reproducable. Tried with two droplets in the same manner, crashes every time. Might be related to a Digital Ocean networking issues, because they had downtime in NYC3, while I'm in AMS3 still experiencing the same problems (droplet comes without ip)

**Comment by andrewsomething** [comment]
Thanks for the report. This was recently addressed in #94 and will be in the next release.

**Comment by andrewsomething** [comment]
I'm happy to say that yesterday, we release version 1.0.0 of the DigitalOcean Terraform provider. It should include a fix for this issue.

You can find the full changelog here: https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/CHANGELOG.md#100-september-27-2018

In order to upgrade an existing project to the latest version, run:

```
terraform init -upgrade
```

If you believe that this issue is being closed by mistake or the problem has not been resolved, please let me know or feel free to open a new issue with additional details.

Thanks for your patience!
